src/pages/Home/Testimonials/index.tsx

```diff
@@ -0,0 +1,66 @@
+import {
+  Badge,
+  Box,
+  Center,
+  Container,
+  Grid,
+  Text,
+  Title,
+} from "@mantine/core";
+import TestimonialBox from "../../../components/TestimonialBox";
+import classes from "./index.module.css";
+
+export default function Testimonials() {
+  const allData = [
```

Comment:
i think we should query this in database

Reply:
riel

Reply:
put that to the backlog